content/en/continuous_integration/quality_gates/_index.md

```diff
@@ -0,0 +1,175 @@
+4. Define the rule condition. The rule condition states in which scenario the rule fails, failing the related pipeline as well. You can select one of the existing rule conditions for the rule type you have selected.
+
+   The following example shows how to create a static analysis rule that will fail when there are one or more static analysis violations with "error" status and "security" category being introduced in a specific commit:
+
+   {{< img src="ci/qg_rule_condition_sa_errors_security.png" alt="Rule for static analysis security errors" style="width:80%;">}}
+
+5. Select whether the rule should block the pipeline or not when it fails. Non-blocking rules are still evaluated, but
+they do not block the pipeline when they fail.
+6. Select a rule name that describes the rule that you are creating.
+7. Click **Save Rule**.
+
+A rule starts being evaluated as soon as it is created.
+
+## Datadog-ci gate evaluate
+
+To use quality gates, [`datadog-ci`][7] version should be higher or equal than `2.16.0`.
+
+You can invoke the Quality Gates evaluation by calling the [`datadog-ci gate evaluate`][4] command.
+
+This command:
+
+1. Retrieves all the impacted rules based on the current pipeline context (branch, repository).
+The rules that are retrieved vary based on the pipeline context, the [rules scopes](#rule-scope) and eventual [custom scopes](#custom-scope).
+2. Evaluates all the impacted rules.
+3. If one or more blocking rules fail, the command fails as well, blocking the pipeline.
+
+<div class="alert alert-danger"><strong>Note:</strong> for the command to work properly, it's important that the events (tests, static analysis violations)
+are sent to Datadog <strong>before</strong> the <code>datadog-ci gate evaluate</code> command is executed.
+Otherwise, the rules might have an incorrect behavior due to the absence of the events.
+</div>
+
+The command requires the `DATADOG_API_KEY` and `DATADOG_APP_KEY` environment variables to point to your [Datadog API Key][5]
+and [Datadog Application Key][6]. Also, you need to set the `DD_BETA_COMMANDS_ENABLED` environment
+variable as `true`. Optionally, you can specify the `DATADOG_SITE` environment variable to point to a specific datadog site.
+The default site is US1 (datadoghq.com).
+
+{{< code-block lang="shell" >}}
+DD_BETA_COMMANDS_ENABLED=true DATADOG_API_KEY=<API_KEY> DATADOG_APP_KEY=<APP_KEY> datadog-ci gate evaluate
+{{< /code-block >}}
+
+The behavior of the command can be modified using the following flags:
+- **--fail-on-empty**: when this flag is specified, the command fails if no matching rules were found in Datadog
+based on the current command scope. By default, the command succeeds.
+- **--fail-if-unavailable**: when this flag is specified, the command fails if one or more rules could not be evaluated.
```

Comment:
```suggestion
- **--fail-if-unavailable**: when this flag is specified, the command fails if one or more rules could not be evaluated because of an internal issue.
```